docs/one-click-integrations/jira-integration.md

```diff
@@ -0,0 +1,42 @@
+# Jira integration
+
+Pulse integrates with Jira Cloud to receive data about issues, necessary to calculate the metrics [Lead time and Cycle time](../metrics/lead-cycle-time.md).
+
+## Setting up the Jira integration
+
+To set up the Jira integration:
```

Comment:
I think we should mention here that for best results, this action should be performed by an admin, since the integration will only have access to the resources that the user installing the app has access to.

Reply:
I added an admonition card stating that the installation step should be done by a Jira administrator. :+1: